directiveAngular/src/app/profilemaking/profilemaking.component.html

```diff
@@ -0,0 +1,65 @@
+<form [formGroup]="ProfileForm" (ngSubmit)="onSubmit()">

```

Comment:
ProfileForm - naming should be profileForm. use pascal case all variables

Reply:
profileForm-this is camel right?

Reply:
yes